### PR TITLE
[OpenStack tutorials] Create /openstack/tutorials page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Create /openstack/tutorials page

**NOTE**: Waiting for tutorials to be uploaded to discourse

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/tutorials or https://ubuntu-com-10687.demos.haus/openstack/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare page against [copydoc](https://docs.google.com/document/d/1_4EJlX-fbcQBVC9Ojhjdbls6kbmITGvHKi5IEqw-HAA/edit#)
- Check tutorials appear in navigation of openstack and are accessible through /openstack/install 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4586

### CODE REVIEW

**Comment by tytus-kurek** [comment]
@petesfrench I'm +1 as long as the link to the "OpenStack upgrades" webinar gets updated.

File: templates/openstack/tutorials.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+          </a>
+        </h3>
+        <p class="u-sv3">Tear down your OpenStack lab environment.</p>
+      </div>
+    </div>                                          
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="u-fixed-width">
```
Comment (minkyngkm) [inline templates/openstack/tutorials.html:213]: Would this be better? referring to the `Learn more...` section in the `/core/smartstart` page. 
```
 <div class="u-fixed-width">
   <h2 class="u-sv2">Additional resources</h2>
 </div>
```

Comment (petesfrench) [reply templates/openstack/tutorials.html:213]: @tytus-kurek what do you think?

Comment (tytus-kurek) [reply templates/openstack/tutorials.html:213]: @petesfrench I'm good with "Learn more"

**Pull request merged**